### ISSUE TITLE
Cannot load sky sphere textures or `.dae` files on CodePen/iPhone 5S

### ISSUE THREAD

**Issue by cvan** [aframevr/aframe-core#707]
Reported by @ericwbailey via https://twitter.com/ericwbailey/status/677577282542809088

CodePen: http://codepen.io/ericwbailey/pen/eJZdXJ

Conversation:

```
‏@ericwbailey: I seem to not be able to view any assets such as a-sky textures or .dae files on CodePen/iPhone 5S. Have others encountered this?
‏       @cvan: do you have a link to a CodePen we can take a look at?
‏@ericwbailey: Sure do: http://codepen.io/ericwbailey/pen/eJZdXJ … Renders just fine on latest Chrome/OSX Yosemite.
       @cvan: renders on my iPhone 6. model was missing on 1st page load but now it's ok. will keep an eye.
@ericwbailey: Nope, still not rendering - sorry if that was unclear.
```

**Comment by dmarcos** [comment]
What iOS version? The Collada Loader doesn't work anymore on iOS after the 9.2 update. Apparently Safari Mobile lowered the number of uniforms you can pass to the shader. The [Three.js example](http://threejs.org/examples/#webgl_loader_collada) is also broken. I already filed  https://github.com/mrdoob/three.js/issues/7807 on three.js

**Comment by dmarcos** [comment]
Also we should have list of phones we support or we will go soon crazy just catching weird bugs.

**Comment by cvan** [comment]
> Also we should have list of phones we support or we will go soon crazy just catching weird bugs.

Agreed; filed as aframevr/aframe-site-dev#83 / aframevr/aframe-site-dev#43. We will add to the repos' READMEs too (perhaps à la https://github.com/substack/browser-badge).

**Comment by ericwbailey** [comment]
Yep, you're correct @dmarcos. I'm on 9.2. I appreciate you looking into this.
